<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRuleDialog</class>
 <widget class="QDialog" name="NewRuleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Rule</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/brick_add.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_2">
        <property name="styleSheet">
         <string notr="true">QLabel { font-size: 14px; }</string>
        </property>
        <property name="text">
         <string>New Rule</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dialogMainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="ruleDetailsGroup">
        <property name="title">
         <string>New Rule Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="ruleNameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="ruleNameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="ruleDirLabel">
           <property name="text">
            <string>Directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="ruleDirLayout">
           <item>
            <widget class="QLineEdit" name="ruleDirEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="ruleDirButton">
             <property name="text">
              <string>Select Directory</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="rulePathLabel">
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="rulePath">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>105</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewRuleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewRuleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ruleNameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewRuleDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ruleDirEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewRuleDialog</receiver>
   <slot>dirChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ruleDirButton</sender>
   <signal>pressed()</signal>
   <receiver>NewRuleDialog</receiver>
   <slot>selectDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nameChanged(QString)</slot>
  <slot>selectDir()</slot>
  <slot>dirChanged(QString)</slot>
 </slots>
</ui>
